model/task/task.go

```diff
@@ -1988,6 +1988,45 @@ func GetTimeSpent(tasks []Task) (time.Duration, time.Duration) {
 	return timeTaken, latestFinishTime.Sub(earliestStartTime)
 }
 
+// GetTaskResultsByVersion gets all tasks for a specific version
+// Query results can be filtered by task name, variant name and status in addition to being paginated and limited
+func GetTaskResultsByVersion(versionID, taskOrVariantName, sortBy string, statuses []string, sortDir, page, limit int) ([]Task, error) {
```

Comment:
I would name this GetTasksByVersion to avoid ambiguity with this returning testresults

Reply:
My point was to avoid using the world "results" in the function name, since that can be confused with test results